plugins/env/app/models/environment_variable.rb

```diff
@@ -24,9 +24,7 @@ class << self
     # preview parameter can be used to not raise an error,
     # but return a value with a helpful message
     # also used by an external plugin
-    def env(deploy, deploy_group, resolve_secrets:, project_specific: nil)
-      env = {}
-
+    def env(deploy, deploy_group, resolve_secrets:, project_specific: nil, env: {})
```

Comment:
maybe `env` should be `defaults` or something? sounds a little odd (also the fact that it's passed in, rather than just being merged in the caller)